src/main/java/net/sf/jabref/model/cleanup/FieldFormatterCleanup.java

```diff
@@ -73,6 +78,17 @@ public FieldFormatterCleanup(String field, Formatter formatter) {
         return fieldChanges;
     }
 
+    private List<FieldChange> cleanupAllTextFields(BibEntry entry) {
+        List<FieldChange> fieldChanges = new ArrayList<>();
+        Set<String> fields = entry.getFieldNames();
+        fields.removeAll(Arrays.asList(FieldName.DOI, FieldName.FILE, FieldName.URL, FieldName.URI));
```

Comment:
Please introduce a class constant ALL_TEXT_FIELDS containing these fields. Please also add ISBN and ISSN.